.github/workflows/build.yml

```diff
@@ -0,0 +1,52 @@
+name: Dockerfile Build
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build-binaries:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Build Container
+        run: |
+          make container-init
+
+      - name: Run Container to Build Binaries
+        run: |
+          docker run --rm -v $(pwd):/fioctl fioctl-build sh -c "git config --global --add safe.directory /fioctl && make build"
```

Comment:
@doanac why do we need this Dockerfile
How are we building the binaries to release? 
Are we using it?

Reply:
the container method was a drive by contribution from someone that didn't have golang on their machine.

Reply:
its not how we build release artificates

Reply:
if no longer useful we could remove it right?